app/src/main/res/values/strings.xml

```diff
@@ -165,31 +165,31 @@
         <item quantity="other">%d minutes</item>
     </plurals>
 
-    <string name="pref_reminder_0minutes_entry">0 minutes</string>
+    <string name="pref_reminder_0minutes_entry">Right away</string>
     <string name="pref_reminder_0minutes_value" translatable="false">0</string>
-    <string name="pref_reminder_5minutes_entry">5 minutes</string>
+    <string name="pref_reminder_5minutes_entry">Five minutes</string>
```

Comment:
I'd argue that using words instead of numbers here makes the menu more cluttered and more difficult to read. It's not prose, but just a settings menu :wink: